boa/src/bigint.rs

```diff
@@ -169,6 +169,15 @@ impl JsBigInt {
             return Err(context.construct_range_error("BigInt negative exponent"));
         };
 
+        let num_bits = (y.to_f64().expect("Unable to convert from BigUInt to f64")
+            * x.inner.bits() as f64)
+            .floor()
+            + 1f64;
```

Comment:
Could you change the order of the operation? I was a bit confused reading `y * x.bits`, I'd say `x.bits * y` is more natural since we're thinking of `x^y`.